doc/tutorials/rt_industry.rst

```diff
@@ -0,0 +1,456 @@
+.. _rt_industry_setup:
+
+How to launch CL-RT VM with ACRN Industry scenario
+##################################################
+
+Verified version
+****************
+
+- Clear Linux version: **30690**
+- ACRN-hypervisor tag: **acrn-2019w31.5-140000p**
+- ACRN-Kernel(Service VM kernel): **acrn-2019w31.5-140000p**
+- ACRN-Kernel(Pre-empt RT kernel): **acrn-2019w31.5-143000p**
+
+Prerequisites
+*************
+Below example is based on Intel Kaby Lake NUC platform with two hard disks,
+SATA disk is for Clear Linux based Service VM and the NVMe disk is for CL-RT VM.
+
+- A Clear Linux host machine (CL ver: 29750) to setup build environment.
```

Comment:
should we change the "host" to "build" or "development"? It's easy to get "host" confused, vs "guest"

Reply:
Now it's changed to ``build machine``.